src/pages/components/accordion/usage.mdx

```diff
@@ -4,20 +4,180 @@ description: The accordion component delivers large amounts of content in a smal
+2. **Icon**: indicates if the panel is open or closed
+3. **Panel**: the section of content associated with an accordion header.
+
+### Alignment
+
+By default the chevron icon is placed on the `end` side of the header. This allows for the title on the `start` side to align with other type elements in the layout.
+
+<DoDontRow>
+<DoDont
+colMd={4} colLg={4}
+caption="Do: use the default icon alignment to keep accordion text aligned with other content on the page.">
+
+![Do: use the default icon alignment to keep accordion text aligned with other content on the page.](./images/accordion-alignment-do.png)
+
+</DoDont>
+<DoDont
+colMd={4} colLg={4}
+type="dont"
+caption="Don’t: place icons on the left and create misalignment with the other text on the page.">
+
+![Don’t: place icons on the left and create misalignment with the other text on the page.](./images/accordion-alignment-dont.png)
+
+</DoDont>
+</DoDontRow>
+
+#### Modifier
+
+In rare scenarios, the accordion can be modified to place the icon in front of the title for only tree-like purposes. However, most instances should use the default alignment and for any pure content or documentation purposes it is required to use the default alignment. Icon placement should be consistent throughout your experience.
+
+### Placement
+
+Accordions can be placed with main page content or placed inside of a container such as a side panel or tile.
+
+<Row>
+<Column colLg={8}>
+
+![accordion placement](images/accordion-placement-2.png)
+
+</Column>
+</Row>
+
+#### Grid placement
+
+When placing accordion on the 2x Grid the indented title aligns to the grid columns and the top and bottom borders hang in the gutter.
 
 <Row>
 <Column colLg={8}>
 
-![accordion example](images/accordion-usage-1.png)
+![accordion grid placement](images/accordion-placement-1.png)
 
 </Column>
 </Row>
+
+## Behavior
+
+### States
+
+The accordion component has two main states: **collapsed** and **expanded**. The chevron icon at the end of the accordion indicates which state the accordion is in. The chevron points down to indicate collapsed and up to indicate expanded.
+
+Accordions begin by default in the collapsed state with all content panels closed. Starting in a collapsed state gives the user a high level overview of the available information.
+
+<Row>
+<Column colLg={8}>
+
+![accordion states](images/accordion-state-1.png)
+
+</Column>
+</Row>
+
+A user can then independently expand each section of the accordion allowing for multiple sections to be open at once.
+
+<Row>
+<Column colLg={8}>
+
+![multiple sections expanded](images/accordion-state-2.png)
+
+</Column>
+</Row>
+
+In addition to the collapsed and expanded states, checkboxes also have interactive states for focus, hover, disabled, and skeleton. See the [style tab](/components/accordion/style) for more details.
+
+<Row>
+<Column colLg={8}>
+
+![accordion interactives states](images/accordion-state-3.png)
+
+</Column>
+</Row>
+
+### Click target
```

Comment:
Would we want to talk through keyboard interaction here too? (same thing just with space/enter but with focus outline instead of click target area)

Reply:
@joshblack Is there an easy way to find out what the existing keyboard commands are for a component?

Reply:
Ok I added some content in, let know me if it looks correct.